wrappers/swift/SemanticContext.swift

```diff
@@ -0,0 +1,131 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Wrapper over `ODWSemanticContext` class that manages the inclusion of semantic context values on logged events.
+public class SemanticContext {
+    private var odwSemanticContext: ODWSemanticContext
```

Comment:
`ODWSemanticContext` is a reference type right? If so, we should change this to `private var context: ODWSemanticContext`

Reply:
i kept the variable name to reflect that it is objc class instance called by the wrapper.